backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,36 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+.admin-nav {
+  @include position(absolute, 0 null 0 0);
+  width: $width-sidebar;
+}
+
+.main-content {
+  margin-left: $width-sidebar;
+}
+
 // Header
 //---------------------------------------------------
-#header {
+.admin-nav-header {
+  padding: 15px;
+  height: 31px * (0.67); // height of .page-title
+  box-sizing: content-box;
   background-color: $color-1;
-  padding: 5px 0;
-}
+  border-bottom: 1px solid $color-border;
+
+  a {
+    display: inline-block;
+    line-height: 64px;
+  }
 
-#logo { height: 40px }
+  img {
+    vertical-align: middle;
+    margin-top: -12px; // hand-tweaking because the solid logo is bottom heavy
```

Comment:
Good call. That corner is bound to be problematic. Either it's a part of the sidebar which "happens" to be the same height as the `.page-title ` or part of the main content with happens to be the same width as the sidebar.

I'm thinking that the latter might help with this positioning problem since the sidebar currently has a static width.